test-app/app/templates/tabs/tabs-with-disable-state.hbs

```diff
@@ -0,0 +1,22 @@
+<div class='w-full max-w-md px-2 py-16 mx-auto'>
+    <TabsGroup class="flex flex-col space-x-1 rounded-xl p-1" @onChange = {{this.onChange}} as |tabs|>
+        <div class='flex space-x-1 rounded-xl bg-sky-600/20 p-1'>
+            {{#each this.titles as |tabName index|}}
+                <tabs.Title
+                    @index={{index}}
+                    @disable={{if (eq index this.disabledToBeIndex) true}}
+                    class='w-full rounded-lg py-2.5 text-sm font-medium leading-5 ring-white ring-opacity-60 ring-offset-2 ring-offset-blue-400 focus:outline-none focus:ring-2 space-x-4 pointer-events-auto {{if (eq index this.activatedIndex) "bg-white shadow"}}'
+                    >
+                {{tabName}}
+                </tabs.Title>
+            {{/each}}
+        </div>
+        <div class="mx-auto w-full max-w-md shadow-md px-4 py-4 rounded-xl bg-white mt-4">
+            {{#each this.contents as |content index|}}
+                <tabs.Content class="content" @content-index={{index}}>
```

Comment:
can we remove the index? and compare via object identity or some other way? using indexes makes looping inefficient / invalidate more frequently than we need it to